src/search/index_manager.h

```diff
@@ -272,6 +273,67 @@ struct IndexManager {
 
     return Status::OK();
   }
-};
 
+  Status FieldValues(engine::Context &ctx, std::string_view index_name, std::string_view input_tag,
```

Comment:
```suggestion
  StatusOr<std::unordered_set<std::string>> FieldValues(engine::Context &ctx, std::string_view index_name, std::string_view input_tag,
```

Reply:
I'm a little confused, how come this is the implementation here, however in other parts of Kvrocks, it uses a ret value? When do I know when to use Status + *ret value compared to using StatusOr<T>?

Reply:
It's preferred to use `StatusOr` instead of `Status` and pointer output.